models/shipping-method/src/shipping-method/shipping-method-draft/presets/sample-data-fashion/index.ts

```diff
@@ -0,0 +1,9 @@
+import dhlShippingMethod from './dhl';
+import upsShippingMethod from './ups';
+
+const presets = {
+  dhlShippingMethod,
+  upsShippingMethod,
```

Comment:
A thought to rename these simply `dhl` and `ups`. We haven't been suffixed these presets with their entity types.